### PR TITLE
Text format refactor

### PR DESCRIPTION
**Pull request by jens-erdmann**
See individual commits for details.

### CODE REVIEW

File: flict/flictlib/format/text_format.py [hunk 1]
```diff
@@ -19,13 +19,11 @@ def __init__(self):
         self.col_size = 18
 
     def format_support_licenses(self, supported_licenses):
-        ret_str = ""
-        for item in supported_licenses:
-            ret_str += " " + str(item) + "\n"
-        return ret_str
+        ret = [f" {item}" for item in supported_licenses]
+        return "\n".join(ret)
```
Comment (priv-kweihmann) [inline flict/flictlib/format/text_format.py:23]: even simpler it would be `return " \n".join(str(item) for item in supported_licenses])`

**Review by priv-kweihmann — APPROVED** [review]
The minor optimization can be done later

File: flict/flictlib/format/text_format.py (hunk 1 above)
Comment (jens-erdmann) [reply flict/flictlib/format/text_format.py:23]: If I understood it correctly, the intend was to have a space in front of the name. " item\n" is the format.

Comment (priv-kweihmann) [reply flict/flictlib/format/text_format.py:23]: so it's `return "\n" + "\n".join(str(item) for item in supported_licenses])` anyway leave it like it is, I'm fine with that

**Pull request merged**